brain/circuit.cpp

```diff
@@ -215,6 +230,12 @@ class MVD3 : public Circuit::Impl
         , _circuit( config.getCircuitSource().getPath( ))
     {}
 
+    size_t getNumNeurons() const final
+    {
+        // Dirty cast until ::MVD3::MVD3File::getNbNeuron becomes const
```

Comment:
That's an issue, not a pull request :) I thought the idea was you fix this one line in MVDTool to avoid the cast in Brion.